src/utils/Api.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const authorization_token = localStorage.getItem('authorization_token');
+
+const instance = axios.create({
+  baseURL: process.env.REACT_APP_BACKEND_URL,
+  headers: {
+    'Access-Control-Allow-Origin': '*',
```

Comment:
This header is repsonse header, not a request header according to https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS, no?